### PR TITLE
change title & download text

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: src/site/artifacts/cargo_dist.rs [hunk 1]
```diff
@@ -6,9 +6,18 @@ use axohtml::elements::{div, li, span};
 use axohtml::{html, text, unsafe_text};
 use cargo_dist_schema::{Artifact, ArtifactKind, DistManifest};
 
+pub fn get_os(name: &str) -> &str {
+    match name {
+        "x86_64-unknown-linux-gnu " => "linux",
+        "x86_64-apple-darwin " => "mac",
+        "x86_64-apple-sillicon " => "mac",
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs]:
```suggestion
        "x86_64-apple-silicon " => "mac",
```

File: src/site/artifacts/cargo_dist.rs [hunk 2]
```diff
@@ -6,9 +6,18 @@ use axohtml::elements::{div, li, span};
 use axohtml::{html, text, unsafe_text};
 use cargo_dist_schema::{Artifact, ArtifactKind, DistManifest};
 
+pub fn get_os(name: &str) -> &str {
+    match name {
+        "x86_64-unknown-linux-gnu " => "linux",
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs]: should this have the space after?

File: src/site/artifacts/cargo_dist.rs [hunk 3]
```diff
@@ -6,9 +6,18 @@ use axohtml::elements::{div, li, span};
...
+        "x86_64-apple-darwin " => "mac",
+        "x86_64-apple-silicon " => "mac",
+        "x86_64-pc-windows-msvc " => "windows",
+        &_ => "",
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs]: we should put something for the base case or a comment that explains why this is not possible

Comment (SaraVieira) [reply src/site/artifacts/cargo_dist.rs]: Done!

File: oranda.json [hunk 4]
```diff
@@ -1,6 +1,5 @@
 {
   "favicon": "https://www.axo.dev/favicon.ico",
-  "path_prefix": "oranda",
```
Comment (ashleygwilliams) [inline oranda.json]: we don't want to remove this do we?

File: src/site/artifacts/cargo_dist.rs [hunk 5]
```diff
@@ -77,26 +86,28 @@ pub fn build(config: &Config) -> Result<Box<div<String>>> {
...
                     targets.push_str(format!("{} ", targ).as_str());
                 }
-                let url = create_download_link(config, &artifact.name);
-                let text = format!("Download v{}", &release.app_version);
                 let install_code = get_install_hint(
                     &release.artifacts,
                     &artifact.target_triples,
                     &config.syntax_theme,
                 );
+                let detect_text = match get_os(targets.as_str()) {
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs:94]: great!

**Review by ashleygwilliams — COMMENTED** [review]
i generally approve but we don't want to remove the prefix

File: oranda.json (hunk 4 above)
Comment (SaraVieira) [reply oranda.json]: shit, fixed

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**